sql/mob_pools.sql

```diff
@@ -2854,7 +2854,7 @@ INSERT INTO `mob_pools` VALUES (2798,'Nahn','Nahn',77,0x000065010000000000000000
 INSERT INTO `mob_pools` VALUES (2799,'Nail_Bomb','Nail_Bomb',236,0x0000FF0600000000000000000000000000000000,1,1,0,0,100,0,0,0,0,0,0,0,0,643,8,0,0,0,0,0,236);
 INSERT INTO `mob_pools` VALUES (2800,'Naja_Salaheem','Naja Salaheem',151,0x0500DB0700000000000000000000000000000000,1,1,0,240,100,0,0,0,0,32,0,32,1,1563,0,0,0,0,0,781,151);
 INSERT INTO `mob_pools` VALUES (2801,'Nakki','Nakki',86,0x0000400100000000000000000000000000000000,1,1,3,270,100,0,1,0,0,6,0,0,0,3,0,0,0,0,0,86,86);
-INSERT INTO `mob_pools` VALUES (2802,'Namorodo','Namorodo',227,0x0000340200000000000000000000000000000000,4,4,7,240,100,0,1,0,0,34,0,32,218,153,0,0,28,1,0,227,227);
+INSERT INTO `mob_pools` VALUES (2802,'Namorodo','Namorodo',227,0x0000340200000000000000000000000000000000,4,4,7,240,100,0,1,0,0,34,0,32,218,131,0,0,28,0,0,227,227);
```

Comment:
What's this adjustment in entityFlags do?

Reply:
Removes two unused flags and sets the second bit - which increases the size from normal skele to medium sized skele

Reply:
![Uploading image.png…]()

Reply:
![image](https://github.com/AirSkyBoat/AirSkyBoat/assets/105882290/1a25c01a-3490-43b7-a17e-429c7e2a7364)